db/migrate/20191113165001_add_application_choices_reject_by_default_at.rb

```diff
@@ -0,0 +1,5 @@
+class AddApplicationChoicesRejectByDefaultAt < ActiveRecord::Migration[6.0]
+  def change
+    add_column :application_choices, :reject_by_default_at, :datetime
```

Comment:
Since it'll be configurable, perhaps also save the number of days here.

Reply:
Yes could do that, you mean as well as the timestamp?